docs/Build/Meson_build.md

```diff
@@ -1,29 +1,61 @@
 # Build and Installation Instructions with Meson
 
+Meson is a modern and fast build system with a lot of features. You can
+find its documentation at [mesonbuild.com](https://mesonbuild.com/).
+
 ## Install Dependencies
 
-Fedora:
-```bash
-dnf install meson qt5-qtbase-devel rtaudio-devel "pkgconfig(jack) help2man"
-```
+=== "Fedora"
 
-Debian/Ubuntu:
-```bash
-apt install meson build-essential qtbase5-dev librtaudio-dev libjack-jackd2-dev help2man
-```
+    ```bash
+    dnf install meson qt5-qtbase-devel rtaudio-devel "pkgconfig(jack)" help2man
+    ```
 
-MacOS with brew (not tested):
-```bash
-brew install meson qt rt-audio jack help2man
-```
+=== "Debian/Ubuntu"
+
+    ```bash
+    apt install meson build-essential qtbase5-dev librtaudio-dev libjack-jackd2-dev help2man
+    ```
+
+=== "MacOS"
+
+    ```bash
+    brew install meson qt rt-audio jack help2man
```

Comment:
sorry, just noticed this. `qt` in homebrew is now Qt 6. Do we support that? If not, then I'd change this to `qt5`

Reply:
I will change this.